src/styles/abstracts/variables.tsx

```diff
@@ -1,5 +1,13 @@
 export const white = "#ffffff";
 export const black = "#000000";
+export const greyDark = "#4F4F4F";
+export const greyLight = "#DDDDDD";
+export const blue = "#0099CC";
+export const blueLighter = "#e6f5fa";
+export const orange = "#FF9220";
+export const greenLight = "#00CBBC";
+
```

Comment:
can shift all these to tailwind config and import from there so we have a common source of truth when we make use of tailwind utilities later on

Reply:
hmm, i have been trying it out, but after i put it in the config file, I cant get the color defined in a styled component unless i use it as a className.